crates/rust-analyzer/src/cli/scip.rs

```diff
@@ -34,6 +32,20 @@ impl flags::Scip {
             prefill_caches: true,
         };
         let root = vfs::AbsPathBuf::assert(std::env::current_dir()?.join(&self.path)).normalize();
+
+        let mut config = crate::config::Config::new(
+            root.clone(),
+            lsp_types::ClientCapabilities::default(),
+            /* workspace_roots = */ vec![],
+        );
+
+        if let Some(p) = self.config_path {
+            let mut file = std::io::BufReader::new(std::fs::File::open(p)?);
+            let json = serde_json::from_reader(&mut file)?;
+            config.update(json)?;
+        }
+        let mut cargo_config = config.cargo();
+        cargo_config.sysroot.get_or_insert(RustLibSource::Discover);
```

Comment:
Isn't this the default?

Reply:
Yeah I guess it is. I wanted to make sure I didn't change the no-config-path version of this, but now that we go through the more general config mechanism we should be getting the defaults properly.